spec/helpers/service_helper_spec.rb

```diff
@@ -222,6 +222,106 @@
     end
   end
 
+  describe '.replace_service_tags' do
+    subject { replace_service_tags content }
+
+    context 'with UK service' do
+      include_context 'with UK service'
+
+      context 'without tags' do
+        let(:content) { 'this is some sample content' }
+
+        it { is_expected.to eql 'this is some sample content' }
+      end
+
+      context 'with SERVICE_NAME tag' do
+        let(:content) { 'You are on the [[SERVICE_NAME]]' }
+
+        it { is_expected.to eql 'You are on the UK Integrated Online Tariff' }
+      end
+
+      context 'with SERVICE_PATH tag' do
+        let(:content) { '<a href="[[SERVICE_PATH]]/browse">Browse</a>' }
+
+        it { is_expected.to eql '<a href="/browse">Browse</a>' }
+      end
+
+      context 'with SERVICE_REGION' do
+        let(:content) { 'within [[SERVICE_REGION]]' }
+
+        it { is_expected.to eql 'within the UK' }
+      end
+
+      context 'with multiple tags' do
+        let :content do
+          <<~END_OF_CONTENT
+            [[SERVICE_NAME]]
+            * [Find commodity]([[SERVICE_PATH]]/find_commodity)
+            * [Browse]([[SERVICE_PATH]]/browse)
+          END_OF_CONTENT
+        end
+
+        let :expected do
+          <<~END_OF_EXPECTED
+            UK Integrated Online Tariff
+            * [Find commodity](/find_commodity)
+            * [Browse](/browse)
+          END_OF_EXPECTED
+        end
+
+        it { is_expected.to eql expected }
+      end
+    end
+
+    context 'with XI service' do
```

Comment:
The tests in this context are almost a identical of the previous ones, wondering if we can dry this a bit. 
If not it's fine, don't stress 😄

Reply:
Yeh - wondered about that _but_ the input and output strings are different each time - I'll see if I can come up with something

Reply:
wrote a `shared_behaviour` but it ended up being 1 line shorter per block and more confusing so I've left it as is